src/script_opt/ScriptOpt.cc

```diff
@@ -275,6 +275,7 @@ static void init_options() {
     check_env_opt("ZEEK_NO_ZAM_OPT", analysis_options.no_ZAM_opt);
     check_env_opt("ZEEK_NO_ZAM_CONTROL_FLOW_OPT", analysis_options.no_ZAM_control_flow_opt);
     check_env_opt("ZEEK_DUMP_ZAM", analysis_options.dump_ZAM);
+    check_env_opt("ZEEK_DUMP_FINAL_ZAM", analysis_options.dump_final_ZAM);
```

Comment:
That's the main motivation (not always easy/feasible), along with the desire to do things like "let's run the whole test suite with control-flow optimization turned off".

Regarding the single variable approach, do you mean a single variable whose text is stuffed onto the command-line (essentially), and then all of the above would be expressible as `-O xyz` options? That seems fine to me, other than making the `-O xyz` options a bit more complicated still

Reply:
Yeah, that's what I meant — and yeah, it would assume that whatever config you need is expressible at the command line. I didn't look in detail to see to which degree that's currently the case. Not urgent.